### DISCUSSION TITLE
Which battery pack to use?

### DISCUSSION THREAD

**Discussion by kevindelord** [MiczFlor/RPi-Jukebox-RFID#2226] [Q&A] [answered]
Hello friends, 

I have a built a cute little Phoniebox for my daughter using a RaspberryPi 3B+ and a power bank. 
Sadly it is very hard to use as both the RPi and the power bank need to be manually turned on. 

Therefore I'm looking for a solution to plug-in a battery pack onto the RPi itself, which when pressing a switch button (via GPIO control) would turn on the PI via the battery. 
Is that even feasible? 

I have the switch button already - and it's working fine, providing the RPi has power. 

Searching online I found the [Pisugar2 Pro](https://www.amazon.de/-/en/Pisugar2-Portable-Platform-Raspberry-Accessories/dp/B08D8PPCKN/?_encoding=UTF8&pd_rd_w=bpbWv&content-id=amzn1.sym.5c5e380f-adae-4188-be75-d3ed6ab75549%3Aamzn1.symc.adba8a53-36db-43df-a081-77d28e1b71e6&pf_rd_p=5c5e380f-adae-4188-be75-d3ed6ab75549&pf_rd_r=K23656R8PXK84KVT4256&pd_rd_wg=T9uyN&pd_rd_r=38d043a3-6cc9-4910-8a29-26d9cbcee32e&ref_=pd_gw_ci_mcx_mr_hp_atf_m) and/or a [Battery Pack from AZDelivery](https://www.amazon.de/-/en/AZDelivery-Portable-Battery-Compatible-Raspberry/dp/B072QZXTGS/?_encoding=UTF8&pd_rd_w=bpbWv&content-id=amzn1.sym.5c5e380f-adae-4188-be75-d3ed6ab75549%3Aamzn1.symc.adba8a53-36db-43df-a081-77d28e1b71e6&pf_rd_p=5c5e380f-adae-4188-be75-d3ed6ab75549&pf_rd_r=K23656R8PXK84KVT4256&pd_rd_wg=T9uyN&pd_rd_r=38d043a3-6cc9-4910-8a29-26d9cbcee32e&ref_=pd_gw_ci_mcx_mr_hp_atf_m&th=1). 
However I'm not sure if the voltage is enough to power everything (RPi 3B+, speakers, RFID) nor how long it would last (is 3800mAh or 5000mAh enough)?

Thanks in advance for your help and advices!

Kevin 

(And obviously thanks for this amazing project!)

**Comment by s-martin** [comment, marked as answer]
Many people use the [OnOffShim](https://shop.pimoroni.com/products/onoff-shim?variant=41102600138).

For instructions see e.g. here https://splittscheid.de/akku-powerbank-einschalter-raspberry-pi/ (Sorry, only German).

**Reply by kevindelord** [reply]
Ok thanks, looks like it should "wake up" my power bank as well. 
I will order one and see how it goes!

**Reply by pabera** [reply]
Just in case you want to learn more about using it. In English ;-)
https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/documentation/builders/components/power/onoff-shim.md

**Reply by kevindelord** [reply]
Oh well, thank you. 
I have ordered one OnOffShim, have bought a soldering kit for that purpose and have put it all together. 
However it looks like I've butchered the "soldering" part and it does not work. 

Am I right to assume that all pins should be soldered on the OnOffShim? 

Cheers,
Kevin

**Reply by kevindelord** [reply]
Alright gentlemen, 
I started all over again and it works now like a charm.
Soldering ain't easy...
Thanks!
